Snippets/Snippets_5/Monitoring/EndpointConfig.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace MonitoringNotifications.ServiceControl.Notifications
+{
+    using System;
+    using NServiceBus;
+    using NServiceBus.Persistence;
+
+    #region MessageFailedEndpointConfig 5.0
+    public class EndpointConfig : IConfigureThisEndpoint, AsA_Server
```

Comment:
how is the JsonSerializer relevant to what you are trying to the point you are trying to illustrate?